src/Services/setContacts.js

```diff
@@ -0,0 +1,17 @@
+import { doc, setDoc } from "firebase/firestore";
+import { fireStore } from "../Config/firebaseInit";
+
+import { v4 as uuidv4 } from 'uuid';
+
+const setContacts = async(form) => {
+    await setDoc(doc(fireStore, "contacts", form.email), {
```

Comment:
in this function the async await is unnecessary, you can just return the function which you called inside this function